src/style.css

```diff
@@ -1 +1,59 @@
-/* Styles go here */
+* {
+	font-family: 'Robot';
+	font-size: 12px;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+header {
+	width: 1200px;
+	min-width: 1024px;
+	height: 60px;
```

Comment:
do not hardcode height. make for links vertical paddings which will make necessary height